diff --git a/node_modules/metro-file-map/src/watchers/FallbackWatcher.js b/node_modules/metro-file-map/src/watchers/FallbackWatcher.js
index e431e90..83d28ca 100644
--- a/node_modules/metro-file-map/src/watchers/FallbackWatcher.js
+++ b/node_modules/metro-file-map/src/watchers/FallbackWatcher.js
@@ -116,13 +116,21 @@ class FallbackWatcher extends _AbstractWatcher.AbstractWatcher {
     if (this.watched[dir]) {
       return false;
     }
-    const watcher = _fs.default.watch(
-      dir,
-      {
-        persistent: true,
-      },
-      (event, filename) => this._normalizeChange(dir, event, filename),
-    );
+    let watcher;
+    try {
+      watcher = _fs.default.watch(
+        dir,
+        {
+          persistent: true,
+        },
+        (event, filename) => this._normalizeChange(dir, event, filename),
+      );
+    } catch (error) {
+      if (isIgnorableFileError(error)) {
+        return false;
+      }
+      throw error;
+    }
     this.watched[dir] = watcher;
     watcher.on("error", this._checkedEmitError);
     if (this.root !== dir) {
